applications/nrf5340_audio/doc/building.rst

```diff
@@ -298,21 +299,21 @@ After building the files for the development kit you want to program, follow the
 When using the default CIS configuration, if you want to use two headset devices, you must also populate the UICR with the desired channel for each headset.
 Use the following commands, depending on which headset you want to populate:
 
-* Left headset (``--val 0``):
+* Left headset (``--value 0``):
 
    .. code-block:: console
 
-      nrfjprog --memwr 0x00FF80F4 --val 0
+      nrfutil device x-write --address 0x00FF80F4 --value 0
 
-* Right headset (``--val 1``):
+* Right headset (``--value 1``):
 
    .. code-block:: console
 
-      nrfjprog --memwr 0x00FF80F4 --val 1
+      nrfutil device x-write --address 0x00FF80F4 --value 1
 
 Select the correct board when prompted with the popup.
-Alternatively, you can add the ``--snr`` parameter followed by the SEGGER serial number of the correct board at the end of the ``nrfjprog`` command.
-You can check the serial numbers of the connected devices with the ``nrfjprog -i`` command.
+Alternatively, you can add the ``--snr`` parameter followed by the SEGGER serial number of the correct board at the end of the ``nrfutil device`` command.
+You can check the serial numbers of the connected devices with the ``nrfutil device list`` command (you need to remove the two initial zeroes from the nRF5340 Audio DK serial number).
```

Comment:
I see that `nrfutil device list`outputs: 
![image](https://github.com/user-attachments/assets/74129346-a5a1-46c9-9871-1db688217650)
So no need to remove any initial zeros
```suggestion
You can check the serial numbers of the connected devices with the ``nrfutil device list`` command).
```